server/routes/viewModels/taskListModels.js

```diff
@@ -159,3 +159,8 @@ function getBassDetails({bassReferralNeeded, bassAccepted, bassWithdrawn}, {bass
         bassOfferMade: bassReferralNeeded && bassOffer === 'DONE' && !bassWithdrawn && !bassExcluded
     };
 }
+
+function versionAheadOfApproved(versionDetails, approvedVersionDetails) {
+    return versionDetails.version > approvedVersionDetails.version ||
```

Comment:
looks fine _provided_ that once they have started varying the licence version will never change i.e. we won't get versions of 2.4 and 4.2 to compare...

Reply:
Agreed